### PR TITLE
add jshint

### PR DESCRIPTION
**Pull request by calvinmetcalf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,12 @@
   "directories": {
     "test": "test"
   },
- "devDependencies": {
-   "karma": "~0.10"
+  "devDependencies": {
+    "karma": "~0.10",
+    "jshint": "^2.4.4"
```
Comment (paulswartz) [inline package.json]: Not sure if this is a typo or something that package.json is supposed to support, but it's breaking the build on Travis.

**Comment by paulswartz** [comment]
Did you mean to include a `.jshintrc`?

**Comment by calvinmetcalf** [comment]
fixed

**Comment by calvinmetcalf** [comment]
I didn't add a .jshintrc because that  kind of stuff is up to you, if you'd like I can drop in one of my goto .jshintrc files.

Also looks like i screwed up the karma link, so fixed (plus you don't need to worry about ./node_modules npm takes care of that)

**Comment by paulswartz** [comment]
I have one at hone home that I've been developing with; I'll clean it up and put it in.

**Pull request merged**